src/game/lara.c

```diff
@@ -2124,14 +2124,12 @@ bool LaraTestHangJumpUp(ITEM_INFO *item, COLL_INFO *coll)
 
     bounds = GetBoundsAccurate(item);
     hdif = coll->front_floor - bounds[FRAME_BOUND_MIN_Y];
-    if (hdif < 0 && hdif + item->fall_speed < 0) {
-        return false;
-    }
+    if (hdif < 0 && hdif + item->fall_speed < 0) { }
```

Comment:
This change looks sketchy, can you double check? If it's needed to be a no-op for the fix to work, let's hide it behind `g_Config` as well

Reply:
I think I screwed up on this change. I pushed all my code because I was switching hard drives and clearly forgot I changed some time. I’ll review it later tonight since I’m busy right now.